src/tools/boxed_rng.rs

```diff
@@ -0,0 +1,23 @@
+use rand_core::{CryptoRng, CryptoRngCore, RngCore};
+
+/// Since object-safe trait methods cannot take `impl CryptoRngCore` arguments,
+/// this structure wraps the dynamic object and exposes a `CryptoRngCore` interface,
+/// to be passed to statically typed round methods.
```

Comment:
```suggestion
/// Since object-safe trait methods cannot take `impl CryptoRngCore` arguments,
/// this structure wraps the dynamic object and exposes a `CryptoRngCore` interface,
/// to be passed to statically typed round methods.
// TODO: remove when the RustCrypto crates are updated (#123456).
```

Reply:
Filed #216